### DISCUSSION TITLE
Blog 5

### DISCUSSION THREAD

**Discussion by DonKir15** [Anastylos/Jump-and-Run#58] [Blog]
## Fortschritte der Woche

In dieser und den letzten Wochen haben wir unser Projekt über den „Hello World“-Stand hinaus entwickelt und bereits viele Elemente integriert, die in unserem Game Design Dokument vorgesehen sind. Statt einer einfachen Demo haben wir einen **"Playground"** erstellt, in dem verschiedene Funktionen ausprobiert und weiterentwickelt werden können. Dieser Ansatz erlaubt uns, Ideen zu testen und Features zu implementieren, die später in die finalen Level integriert werden.

### Einblicke in unsere Demo

Hier ein paar Eindrücke unserer Fortschritte:

![Demo Feuerpfeil](https://github.com/user-attachments/assets/f94a1efd-7aac-4673-81b5-6ce8fc39df14)

![Pfeil in Wand](https://github.com/user-attachments/assets/0e8352ac-692d-4d8b-928f-9c5c80c6c375)

![Tod im Wasser](https://github.com/user-attachments/assets/8eddf727-ec8e-477e-8dcb-43a2ced8519f)

## Aufgabenverteilung und Mitwirkende

- **Game Design & Level Testing:** Don, Dinis, Lukas, Christoph
- **Implementierung der Grundmechaniken:** Michael, Christoph
- **Technische Umsetzung und Fehlerbehebung:** Michael, Christoph

**Comment by ldcdorn** [comment]
Wow :astonished:, euer Team hat ja schon massiv vortschritt gemacht, das ist echt viel für eine erste Demo. Da bekommt man gleich Lust, das ganze selbst zu spielen. Uns würde interessieren, wie ihr das ganze mit dem Artwork gemacht habt. Habt ihr die Assets alle selber entwickelt oder ist das ein vorgefährtigtes Asset-pack? Auch eine Sehr gute Idee, das ganze mit GIFs zu zeigen, da kriegt man gleich einen viel besseren Eindruck von dem Gameplay, als man das bei statischen Bildern hätte.

Viele Grüße
Project HÆLTH

**Reply by JustMischka** [reply]
Hallo ldcdorn,

vielen Dank für deinen freundlichen Kommentar! 😊
Wir haben bisher vorgefertigte Asset-Packs genutzt, um uns voll auf ein reibungsloses Gameplay konzentrieren zu können. 
Unser Ziel ist es, ein möglichst angenehmes Spielerlebnis zu schaffen.

Viele Grüße
Team Jump and Run

**Comment by schmid-di** [comment]
Es klingt, als ob euer Projekt tolle Fortschritte macht! Der Wechsel von einer einfachen Demo zu einem "Playground" ist eine clevere Idee, die viel Flexibilität schafft und Raum für kreatives Experimentieren gibt. So könnt ihr gezielt verschiedene Funktionen ausprobieren und optimieren, bevor sie ins finale Spiel eingebaut werden. Ein wertvoller Schritt, um die Qualität des Endprodukts zu sichern!

Euer UMOC-Team

/JM /PL
